packages/edit-site/src/store/selectors.js

```diff
@@ -88,18 +88,18 @@ export const getSettings = createSelector(
  *
  * @param {Object} state Global application state.
  *
- * @return {number} Home template ID.
+ * @return {number?} Home template ID.
  */
 export function getHomeTemplateId( state ) {
-	return state.homeTemplateId;
+	return state.home.templateId;
 }
 
 /**
  * Returns the current template ID.
  *
  * @param {Object} state Global application state.
  *
- * @return {number} Template ID.
+ * @return {number?} Template ID.
```

Comment:
Wondering which underlying change is allowing for this to be `undefined`. Shouldn't there always be a 'current' template?

Reply:
This was already the case, but the template ID can be undefined since we load the template async at the beginning. Additionally, if anything errors out, we may end up without a "current template" (e.g. that "empty document" situation). So I wanted to change the type so that we handle this scenario safely without crashing